lib/plugins/track.js

```diff
@@ -189,7 +198,7 @@ exports.configure = function configure (config) {
 
         // Break when cookies aren't being cleared and it gets set properly
         // Don't break when uuid is falsy so all the cookies get cleared
-        if (cookie.getItem(storage.name) && uuid) {
+        if (uuid && cookie.getItem(storage.name) === uuid) {
```

Comment:
here isn't very clear (damn operator precedence :( ), perhaps wrap `()` around the latter part of the expression